test/unit/qrda/cat3_test.rb

```diff
@@ -0,0 +1,32 @@
+require_relative '../../test_helper'
+require 'cqm/models'
+require 'cqm_validators'
+
+module QRDA
+  module Cat3
+    class Cat3CVUTest < MiniTest::Test
```

Comment:
I would change the name of the class to be more generic.  'CVU' is a little to context specific for this library.

Name it something like Cat3ValidationTest

Reply:
I've changed the class name to Cat3ValidationTest.